jogoDaVelha.html

```diff
@@ -0,0 +1,224 @@
+        text-shadow:0px 1px 0px #528009;
+        align-content: center;
+        border-radius: 10px;
+        font-family: 'Luckiest Guy', cursive;
+        font-size: 25px;
+    }
+</style>
+
+<script>
+    //VARIAVEIS ESCOPO GERAL
+    var pGeral = document.getElementsByClassName ('quadrado');
+    var popUpVitoria = document.getElementById('popUpVitoria');
+    var popUpEmpate = document.getElementById('popUpEmpate');
+    var placarX = document.getElementById('jogadorX');
+    var placarO = document.getElementById('jogadorO');
+    var vitoria = document.getElementById('voceGanhou');
+    var clique = false
+    var numeroO = 0
+    var numeroX = 0
+    // REINICIAR PARTIDA
+    function reiniciar () { 
+        for ( var i = 0 ; i < pGeral.length ; i++) {
+            pGeral[i].textContent = ""
+            popUpVitoria.className = 'popUpVitoriaFechado' ;
+            popUpEmpate.className = 'popUpEmpateFechado' ;
+        }
+    }
+    //PLACAR
+    function adicionaPlacar () {
+        if (clique) { 
+            numeroX ++
+            placarX.textContent = "Vitórias jogador X = " + (numeroX);
+            alertaVitoria("Xis");
+        } else { 
+            numeroO ++
+            placarO.textContent = "Vitórias jogador O = " + (numeroO);
+            alertaVitoria("bola");
+        }
+    }
+    function zeraPlacar () {
+        for ( var i = 0 ; i < pGeral.length ; i++) {
+            if(pGeral[i].textContent != "") {
+                return 
+            }   
+        }
+        numeroO = 0;
+        numeroX = 0;
+        placarO.textContent = "Vitórias jogador O = 0";
+        placarX.textContent = "Vitórias jogador X = 0";
+    }
+    //ALERTAS 
+    function alertaVitoria (jogador) {
+        setTimeout(function() {
+          vitoria.textContent = "Jogador "+ jogador + " ganhou";
+          popUpVitoria.className = 'popUpVitoriaAberto'
+        },50) 
+    }
+    function alertaEmpate () {
+        for ( var i = 0 ; i < pGeral.length ; i++){
+            if(pGeral[i].textContent == "" ) {
+                return 
+            }
+        }
+        setTimeout(function() {
+            popUpEmpate.className = 'popUpEmpateAberto' ;
+        },50) 
+    }
+    //AÇÃO DO USUARIO
+    function clicarQuadrado (id) {
+        elemento = document.getElementById (id);
+        vezJogador = document.getElementById ('vezJogador')
+        if (popUpVitoria.className == 'popUpVitoriaFechado') {
+            if (elemento.textContent == "") {
+                if (clique == false) {
+                    elemento.textContent = "X";
+                    vezJogador.textContent = "PRÓXIMO = BOLA"    
+                } else {
+                    elemento.textContent = "O";
+                    vezJogador.textContent = "PRÓXIMO = XIS"
+                }   
+                clique = !clique
+                if (verificaVitoria ()) {
+                    adicionaPlacar()
+                }else {
+                    alertaEmpate()
+                }
```

Comment:
me confundi um pouco aqui hehehehe.
seria melhor criar um verificaEmpate, ficaria mais legivel
```suggestion
                } else if (verificaEmpate()) {
                    exibeAvisoEmpate();
                }
```